templates/taxi/car_detail.html

```diff
@@ -12,11 +12,30 @@
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+    {% if user in car.drivers.all %}
+      <form action = "{% url 'taxi:toggle-driver' %}" method="post">
+        {% csrf_token %}
+        <input  type="hidden" name="action" value="remove">
+        <input  type="hidden" name="car_id" value="{{ car.id }}">
+        <button class="btn btn-danger link-to-page" type="submit">Delete me from this car</button>
+      </form>
+    {% else %}
+      <form action = "{% url 'taxi:toggle-driver' %}" method="post">
+        {% csrf_token %}
+        <input  type="hidden" name="action" value="add">
+        <input  type="hidden" name="car_id" value="{{ car.id }}">
+        <button class="btn btn-success link-to-page" type="submit">Assign me to this car</button>
+      </form>
+    {% endif %}
+  <h1>
+    Drivers  
+  </h1>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
```

Comment:
Consider using `car.drivers.all()` with parentheses to explicitly call the method, which is the correct syntax for Django queryset evaluation.